client/js/_author/components/assessments/preview_container.jsx

```diff
@@ -7,13 +7,24 @@ export default class PreviewContainer extends React.Component {
     getAssessmentOffered: React.PropTypes.func.isRequired,
     assessmentPlayerUrl: React.PropTypes.string.isRequired,
     apiUrl: React.PropTypes.string.isRequired,
+    authoringToolPreviewSettings: React.PropTypes.arrayOf(React.PropTypes.string).isRequired,
   };
 
   static hasOffered(assessment) {
     return !(_.isUndefined(assessment.assessmentOffered)
       || _.isEmpty(assessment.assessmentOffered[0]));
   }
 
+  static buildEmbedUrl(props) {
```

Comment:
It seems unnecessary to have a static method that actually only uses instance params, and it should be about a minute to switch it back.